### PR TITLE
Lakeshore 325: adding two attri: resistance and heater_output

### PR DESCRIPTION
**Pull request by liangosc**
Changes proposed in this pull request:
- In Lakeshore/Model_325.py  a new parameter "resistance" was added to obtain the resistance of the system
- In Lakeshore/Model_325.py, a new parameter "heater_output" was added to get the heater output (in %) of the system
- added examples to driver_examples/Qcodes example with Lakeshore 325.ipynb for the two new parameters
- minor formatting issue
@sohailc

### CODE REVIEW

File: qcodes/instrument_drivers/Lakeshore/Model_325.py [hunk 1]
```diff
@@ -472,6 +472,24 @@ def __init__(self, parent: 'Model_325', name: str, loop: int) ->None:
             get_cmd=f"RAMPST? {self._loop}"
         )
 
+        self.add_parameter(
+            "resistance",
+            get_cmd=f"HTRRES? {self._loop}",
+            set_cmd=f"HTRRES {self._loop} {{}}",
+            vals=Enum(25, 50),
```
Comment (astafan8) [inline qcodes/instrument_drivers/Lakeshore/Model_325.py]: enumeration is not needed once `val_mapping` is used
```suggestion
```

File: qcodes/instrument_drivers/Lakeshore/Model_325.py [hunk 2]
```diff
@@ -472,6 +472,24 @@ def __init__(self, parent: 'Model_325', name: str, loop: int) ->None:
             get_cmd=f"RAMPST? {self._loop}"
         )
 
+        self.add_parameter(
+            "resistance",
```
Comment (astafan8) [inline qcodes/instrument_drivers/Lakeshore/Model_325.py]: please also add a `label`

File: qcodes/instrument_drivers/Lakeshore/Model_325.py [hunk 3]
```diff
@@ -472,6 +472,24 @@ def __init__(self, parent: 'Model_325', name: str, loop: int) ->None:
...
+            vals=Enum(25, 50),
+            val_mapping={
+                25: 1,
+                50: 2,
+            },
+            unit="Ohm"
+        )
+
+        self.add_parameter(
+            "heater_output",
```
Comment (astafan8) [inline qcodes/instrument_drivers/Lakeshore/Model_325.py]: please also add a `label` and `unit`

File: docs/examples/driver_examples/Qcodes example with Lakeshore 325.ipynb [hunk 4]
```diff
@@ -3645,6 +3645,76 @@
...
+       "0.0"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "lake.heater_1.heater_output() # in %, 50 means 50%"
```
Comment (astafan8) [inline docs/examples/driver_examples/Qcodes example with Lakeshore 325.ipynb]: please add a clarification as in the instrument's manual: smth around the lines of "heater output in percent (%) of current or power, depending on setting."

File: qcodes/instrument_drivers/Lakeshore/Model_325.py (hunk 1 above)
Comment (liangosc) [reply qcodes/instrument_drivers/Lakeshore/Model_325.py]: All the lines with "vals=Enum(...)" followed by "val_mapping={...}" have been removed. (not only for the two new parameters, but also for a few old ones)

File: qcodes/instrument_drivers/Lakeshore/Model_325.py (hunk 2 above)
Comment (liangosc) [reply qcodes/instrument_drivers/Lakeshore/Model_325.py]: Label "Resistance" has been added.

File: qcodes/instrument_drivers/Lakeshore/Model_325.py (hunk 3 above)
Comment (liangosc) [reply qcodes/instrument_drivers/Lakeshore/Model_325.py]: Label "Heater Output" and unit "%" have been added.

File: docs/examples/driver_examples/Qcodes example with Lakeshore 325.ipynb (hunk 4 above)
Comment (liangosc) [reply docs/examples/driver_examples/Qcodes example with Lakeshore 325.ipynb]: Clarification has been added, together with instruction how to get the setting value.

**Pull request merged**